### ISSUE TITLE
Addition to your list of networked notebooks

### ISSUE THREAD

**Issue by uxsdi** [prathyvsh/networked-notebooks#6]
Please consider adding https://dendron.so/ to your list. Hierarchical note system which is being actively developed

**Comment by prathyvsh** [comment]
Thanks for the note! Adding it in.

**Comment by viaahmed** [comment]
Does _kumu.io_ fit into the types you are collecting?

**Comment by prathyvsh** [comment]
> Does _kumu.io_ fit into the types you are collecting?

Yeah, definitely. I had tried Kumu a while back and definitely thought it was one of the more polished websites for creating graphs!
